src/ports/postgres/modules/graph/wcc.py_in

```diff
@@ -117,12 +118,21 @@ def wcc(schema_madlib, vertex_table, vertex_id, edge_table, edge_args,
     old_new_update_where_condition = ''
     new_to_update_where_condition = ''
     edge_to_update_where_condition = ''
+    edge_inverse_to_update_where_condition = ''
 
     INT_MAX = 2147483647
     component_id = 'component_id'
     grouping_cols_comma = '' if not grouping_cols else grouping_cols + ','
     comma_grouping_cols = '' if not grouping_cols else ',' + grouping_cols
 
+    if not is_platform_pg():
+        plpy.execute("DROP TABLE IF EXISTS {edge_inverse}".format(**locals()))
```

Comment:
We shouldn't drop a table we did not create. This drop should be at the end of the function instead.